manifest-rhel-9.6.yaml

```diff
@@ -0,0 +1,74 @@
+# Manifest for RHCOS based on RHEL 9.6
+
+rojig:
+  license: MIT
+  name: rhcos
+  summary: RHEL CoreOS 9.6
+
+variables:
+  osversion: "rhel-9.6"
+
+# Include manifests common to all RHEL and CentOS Stream versions
+include:
+  - common.yaml
+
+# CentOS Stream 9 repos for now
+repos:
+  - c9s-baseos
+  - c9s-appstream
+
+# Eventually we should try to build these images as part of the RHEL composes.
+# In that case, the versioning should instead be exactly the same as the pungi
+# compose ID.
+automatic-version-prefix: "9.6.<date:%Y%m%d%H%M>"
+# This ensures we're semver-compatible which OpenShift wants
+automatic-version-suffix: "-"
+
+mutate-os-release: "9.6"
+
+repo-packages:
+  - repo: c9s-baseos
+    packages:
+ # We include the generic centos release package and fake the red hat os-release
+ # info in a post-process script
+     - centos-stream-release
+
+# Fake out RHEL version in the os-release while waiting for RHEL-9.6 release.
```

Comment:
should we just be find/replace `9.4` with `9.6` in the `os-release` here? i.e. in `manifest-rhel-9.4.yaml` there is no postprocess of `os-release` that is done now (since 4e09d27) so we shouldn't need to replace the whole thing?

Reply:
> should we just be find/replace 9.4 with 9.6 in the os-release here?

Doing that would require installing `redhat-release` from the rhel 9.4 repos. That release file comes from `centos-strea-release`. I tried to do that, as that would make things simpler here but then conflicts with `centos-stream-release` which is required to get the RPM repository files and GPG keys and causes issues when trying to build the extensions container (which needs to downloads RPMs)